coordinator/src/node/expired_positions.rs

```diff
@@ -33,70 +36,88 @@ pub async fn close(node: Node) {
+                    .iter()
+                    .map(|m| QuantityPrice {
+                        quantity: m.quantity,
+                        execution_price: m.execution_price,
+                    })
+                    .collect();
+
+                let closing_price = average_execution_price(amounts)
+                    .to_f32()
+                    .expect("to fit into f32");
+                db::positions::Position::set_open_position_to_closing(
+                    &mut conn,
+                    position.trader.to_string(),
+                    closing_price,
+                )?;
                 continue;
-            }
-            Err(e) => {
-                tracing::error!(?position, "Failed to decide trade action. Error: {e:?}");
+            } else {
+                tracing::trace!(trader_id, order_id, "Skipping expired position as match has already been found. Waiting for trader to come online to execute the trade.");
                 continue;
             }
-        };
+        }
 
-        let closing_price =
-            match BitmexClient::get_quote(&node.inner.network, &position.expiry_timestamp).await {
-                Ok(quote) => quote.get_price_for_direction(position.direction.opposite()),
-                Err(e) => {
-                    tracing::warn!(
-                        "Failed to get quote from bitmex for {} at {}. Error: {e:?}",
-                        position.trader,
-                        position.expiry_timestamp
-                    );
-                    continue;
-                }
-            };
+        tracing::debug!(trader_pk=%position.trader, %position.expiry_timestamp, "Attempting to close expired position");
 
-        // Upon collab closing an expired position we cannot charge a fee using an
-        // invoice. This dummy hash exists in the database to
-        // represent zero-amount invoices.
-        let zero_amount_payment_hash_dummy = PaymentHash(
-            <[u8; 32]>::from_hex(
-                "6f9b8c95c2ba7b1857b19f975372308161fedf50feb78a252200135a41875210",
-            )
-            .expect("static payment hash to decode"),
-        );
+        let new_order = NewOrder {
+            id: uuid::Uuid::new_v4(),
+            // todo(holzeis): we should not have to set the price for a market order. we propably
+            // need separate models for a limit and a market order.
+            price: Decimal::ZERO,
+            quantity: Decimal::try_from(position.quantity).expect("to fit into decimal"),
+            trader_id: position.trader,
+            direction: position.direction.opposite(),
+            order_type: OrderType::Market,
+            // This order can basically not expire, but if the user does not come back online within
+            // 8 weeks I guess we can safely assume that this channel is abandoned and we should
+            // consider force closing.
+            expiry: OffsetDateTime::now_utc().add(Duration::weeks(8)),
```

Comment:
8 weeks is probably too long, we should force close sooner because of the CET's expiry but I understand where you want to get with this.

Reply:
what would be a better value?

Reply:
I'm not sure what our refund timeout is atm, but probably something like `refund_timeout-x` where x is a small number.

Reply:
@luckysori maybe you can help us out here? what is the refund timeout atm?